msteams-platform/bots/how-to/bots-filesv4.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Send and receive files through the bot
+description: Describes how to send and receive files through the bot
+keywords: teams bots files send receive
+ms.date: 05/20/2019
+ms.topic: how-to
+---
+# Send and receive files through the bot
+
+> [!IMPORTANT]
+> The articles in this document are based on the v4 Bot Framework SDK.
+
+There are two ways to send files to and receive files from a bot:
+
+* Using the Microsoft Graph APIs. This method works for bots in all Microsoft Teams scopes:
+  * `personal`
+  * `channel`
+  * `groupchat`
+
+* Using the Teams bot APIs. These only support files in `personal` context.
+
+## Using the Graph APIs
+
+Post messages with card attachments that refer to existing SharePoint files, using the Graph APIs for [OneDrive and SharePoint](/onedrive/developer/rest-api/). To use the Graph APIs, obtain access to either of the following through the standard OAuth 2.0 authorization flow:
```

Comment:
Please recheck the file path.
[OneDrive and SharePoint](/onedrive/developer/rest-api/)

Reply:
Checked file path. The link is working in the docs.microsoft.com site. Not making any changes.